test/e2e/page-objects/PageBackend.js

```diff
@@ -0,0 +1,80 @@
+import { Selector } from "testcafe";
+
+export default function (t) {
+    return {
+        /**
+         * use the navbar to visit the 'Backend'-Page. 
+         * @returns Promise for the Page-Object of 'Backend'-Page
+         */
+        async navigateToViaNavbar() {
+            await t.click("#nav_backend > a");
+            return {
+                /**
+                 * There is a Tab-View with "Ausstellungen", "Gemälde", "News". Via
+                 * the chooseXYZ-Functions you can get the Page-Object for the choosen tab.
+                 */
+                tabs: {
+                    /**
+                     * Choose the Tab 'Ausstellungen' and get its Page-Object
+                     * @returns Promise for the Page-Object
+                     */
+                    async chooseAusstellungen() {
+                        await t.click("#tab_exhibitions > a");
+                        return {
```

Comment:
Maybe you could extract this object the function returns and give it a clear name?

Example:
```
function createTabAusstellungen(t) {
    return {
        async checkAusstellungWirdAngezeigt(name) { /* ... */ },
        async clickNeueAustellung() { /* ... */ },
         { /* ... */ }
    }
}
```

Then you can use it in `navigateToViaNavbar()` like this:
```
async navigateToViaNavbar() {
    await t.click("#nav_backend > a");
    return createTabAusstellungen(t);
}
```

I think this should work 🤔 

Maybe this would make this whole block a bit clearer?